hubblestack/utils/signing.py

```diff
@@ -272,27 +283,28 @@ class X509AwareCertBucket:
     is valid, that the signature was generated by the given public.crt and that
     the public.crt is signed by the ca.crt.
     """
-    PublicCertObj = namedtuple('PublicCertObj', ['crt', 'txt', 'status'])
-    public_crt = tuple()
 
+    PublicCertObj = namedtuple("PublicCertObj", ["crt", "txt", "status"])
+    public_crt = tuple()
 
     def authenticate_cert(self):
-        if any( i.status == STATUS.FAIL for i in self.public_crt ):
+        if any(i.status == STATUS.FAIL for i in self.public_crt):
             return STATUS.FAIL
-        if all( i.status == STATUS.VERIFIED for i in self.public_crt ):
+        if all(i.status == STATUS.VERIFIED for i in self.public_crt):
             return STATUS.VERIFIED
         return STATUS.UNKNOWN
 
     def __init__(self, public_crt=None, ca_crt=None, extra_crt=None):
         try:
             import hubblestack.pre_packaged_certificates as HPPC
+
             # iff we have hardcoded certs then we're meant to ignore any other
             # configured value
-            if hasattr(HPPC, 'public_crt'):
-                log.debug('using pre-packaged-public_crt')
+            if hasattr(HPPC, "public_crt"):
+                log.debug("using pre-packaged-public_crt")
                 public_crt = HPPC.public_crt
-            if hasattr(HPPC, 'ca_crt'):
-                log.debug('using pre-packaged-ca_crt')
+            if hasattr(HPPC, "ca_crt"):
+                log.debug("using pre-packaged-ca_crt")
                 ca_crt = HPPC.ca_crt
         except ImportError:
             pass
```

Comment:
should this really pass silently?

Reply:
i'm not sure this should be fixed in a PR addressing a totally different thing. otherwise, fair point

Reply:
@grogsaxle should we fix this in a different PR, or since we're here might as well?